src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -36,7 +36,7 @@ namespace OpenTelemetry.Instrumentation.Http.Implementation
     /// </remarks>
     internal static class HttpWebRequestActivitySource
     {
-        public const string ActivitySourceName = "OpenTelemetry.HttpWebRequest";
+        public const string ActivitySourceName = "OpenTelemetry.Instrumentation.HttpWebRequest";
```

Comment:
Just tossing one more idea out. When an instrumentation library defines multiple activity sources like, we could adopt the pattern of `AssemblyName`[.{`SomeAdditionalIdentifier`}]. The `SomeAddionalIdentifier` is optional and would only be used when there are multiple activity sources.

So in this case the two ActivitySources in this library would be:

* `OpenTelemetry.Instrumentation.Http.HttpClient` and
* `OpenTelemetry.Instrumentation.Http.HttpWebRequest`

Reply:
@open-telemetry/dotnet-approvers, what do you think? I will be glad to prepare changes, but I would like to have agreement on that. :)

Reply:
I think this is a good idea.

Reply:
agree

Reply:
with .NET 7, we wont control the ActivitySource name, as httpclient already creates activity with its ActivitySource.

Reply:
Fixed, for currently supported set of .NET version.

I think that change will be useful as long OTel support .NET 6 and .NET Framework.